packages/web/src/components/PopupModal/RemindModal.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { PopUpModal } from '.';
+
+const subscribeUser = async () => {
+  const res = await fetch('/api/subscription/subscribe', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+  if (!res.ok) {
+    throw new Error();
+  }
+};
+
+export const RemindMeModal: React.FC = () => (
```

Comment:
This file should be named the same as the component, and we should probably move this out into it's own folder in `components` since this file isn't used in the `PopupModal` component